atlp-rwanda/e-commerce-lydia-32-bn
# Login with email and password

### What does this PR do?

Implement user login functionality.

### Description of Task to be completed?

1. Add user login endpoint to authenticate users.
2. Validate user credentials (email and password).
3. Check if the user is verified.
4. Generate and return a JWT token upon successful authentication.

### How should this be manually tested?

- Make a POST request to the login endpoint with valid email and password.
- Check the response for a success message and a JWT token.

### Any background context you want to provide?

Only users who are verified can log in so as to enhance the overall security, reliability, and integrity of the application.
also comments have been added to the code for clear understanding and maintenance.

### What are the relevant pivotal tracker stories

#187570865: As a user, I can log into the E-commerce application via email & password
